openpype/modules/kitsu/plugins/publish/integrate_kitsu_note.py

```diff
@@ -27,9 +28,19 @@ def process(self, context):
 
             # Get note status, by default uses the task status for the note
             # if it is not specified in the configuration
-            note_status = kitsu_task["task_status"]["id"]
+            shortname = kitsu_task["task_status"]["short_name"].upper()
+            note_status = kitsu_task["task_status_id"]
 
-            if self.set_status_note:
+            # Check if any status condition is not met
+            allow_status_change = True
+            for status_cond in self.status_conditions:
+                condition = status_cond["condition"] == "equal"
+                match = status_cond["short_name"].upper() == shortname
+                if match and not condition or condition and not match:
+                    allow_status_change = False
+                    break
```

Comment:
I propose to turn it into validation to simplify
```suggestion
            allow_status_change = False
            for status_cond in self.status_conditions:
                condition = status_cond["condition"] == "equal"
                match = status_cond["short_name"].upper() == shortname
                if match and condition:
                    allow_status_change = True
                    break
```

Reply:
Okay, my proposal is not good and stops if we don't set any condition, while we would expect it to pass if no any condition is set. Keep it your way.